code/game/objects/items/devices/traitordevices.dm

```diff
@@ -40,7 +40,7 @@
 		spawn((wavelength+(intensity*4))*10)
 			if(M)
 				if(intensity >= 5)
-					M.apply_effect(round(intensity/1.5), PARALYZE)
+					M.Paralyse(intensity * 40/3)
```

Comment:
what's going on here?

Reply:
apply_effect() is only used when you want the effect time to be reduced by armour.
the maths change is for converting cycles to deciseconds.
x /1.5 * 20 = 40x /3